fhirclient/models/activitydefinition.py

```diff
@@ -382,6 +382,7 @@ def elementProperties(self):
 from . import duration
 from . import expression
 from . import fhirdate
+from . import fhirdate
```

Comment:
I think the generating code could do something smart to avoid this double import, but I didn't bother.

Reply:
leave that for a 'move the imports to the top' kind of PR